live-build/variants/internal-dcenter/package-lists/dcenter.list.chroot

```diff
@@ -24,6 +24,7 @@ libldap2-dev
 libsasl2-dev
 nfs-kernel-server
 openjdk-11-jdk-headless
+openjdk-17-jdk-headless
```

Comment:
Do we need both Java 11 and Java 17 on dcol?

Reply:
yes dcoa build still uses java11 , for jenkins use we are installing java17